src/test/kotlin/uk/gov/dluhc/printapi/mapper/CertificateToPrintRequestMapperTest.kt

```diff
@@ -30,15 +36,26 @@ import java.time.Instant
 import java.time.LocalDate
 import java.time.ZoneOffset.UTC
 import java.util.stream.Stream
+import uk.gov.dluhc.printapi.printprovider.models.PrintRequest.CertificateFormat as PrintRequestCertificateFormat
+import uk.gov.dluhc.printapi.printprovider.models.PrintRequest.CertificateLanguage as PrintRequestCertificateLanguage
 
+@ExtendWith(MockitoExtension::class)
 class CertificateToPrintRequestMapperTest {
 
     private lateinit var mapper: CertificateToPrintRequestMapperImpl
 
+    @Mock
+    private lateinit var supportingInformationFormatMapper: SupportingInformationFormatMapper
+
+    @Mock
+    private lateinit var certificateLanguageMapper: CertificateLanguageMapper
+
     @BeforeEach
     fun setup() {
```

Comment:
Hi @NathanRussellValtech ,
One suggestion. I think if we use `@InjectMocks` instead of setting the dependencies with reflection it will be nicer and we won't need to use the actual `InstantMapper` as well, since it is already tested.

<img width="1911" alt="image" src="https://user-images.githubusercontent.com/39367710/204763409-da68db98-cfcb-427b-84fa-976ef23947a3.png">

<img width="1747" alt="image" src="https://user-images.githubusercontent.com/39367710/204763643-b53b1a62-f4db-4c1c-b59e-1a1c4943f4c3.png">

Reply:
Good shout; yes, I've done that. Thanks 👍